inputs/values/azure_dockers.toml

```diff
@@ -0,0 +1,34 @@
+[images]
```

Comment:
My pet peeve against the word `dockers`, can we call it `containers`?
Do we need this as a base? Or can the original base just be an empty file so the first build pushes both?

Reply:
Yeah, I hate `dockers` a lot... `images.toml` is the naming convention we're already using, so probably `cpg-config/templates/images/gatk-sv/{gcp/azure}_images.toml`?

Reply:
Empty file is an interesting one... I'll ping this workflow with a couple of dry runs to see what happens if the 'dockers' file is empty, or if the file has all the correct keys but None for 'current image' values. I think I understand the behaviour, but I'm keen to confirm